### DISCUSSION TITLE
Error on section 1.8 Foundry Fundamentals

### DISCUSSION THREAD

**Discussion by ROBMORILLO65** [Cyfrin/foundry-full-course-cu#2061] [Q&A]
Hi everyone. I'm new to all this, as some of you, I'm trying hard to learn. I'm getting the errors I'd describe below and I don't know how to fix it. I'd would appreciate your help on this issue.

whoami@DESKTOP-U3OITC2:/mnt/c/Users/Usuario/foundry-f23/foundry-simple-storage-f23$ forge init --force
Target directory is not empty, but `--force` was specified
Initializing /mnt/c/Users/Usuario/foundry-f23/foundry-simple-storage-f23...
Error: 
failed to commit (code=Some(128), stdout="", stderr="Author identity unknown\n\n*** Please tell me who you are.\n\nRun\n\n  git config --global user.email \"you@example.com\"\n  git config --global user.name \"Your Name\"\n\nto set your account's default identity.\nOmit --global to set the identity only in this repository.\n\nfatal: empty ident name (for <whoami@DESKTOP-U3OITC2.>) not allowed")
whoami@DESKTOP-U3OITC2:/mnt/c/Users/Usuario/foundry-f23/foundry-simple-storage-f23$

**Comment by ROBMORILLO65** [comment]
Auto answer.
I'd already found the error, I was writing a wrong user name and a wrong email.

**Reply by EngrPips** [reply]
Interesting, I am glad you unblock yourself. Keep crushing it.
